src/main/kotlin/com/mparticle/kits/ApptentiveKit.kt

```diff
@@ -159,134 +194,83 @@ class ApptentiveKit : KitIntegration(), KitIntegration.EventListener, CommerceLi
             )
         )
         return messages
+    } //endregion
+
+    //region IdentityListener
+    override fun onIdentifyCompleted(
+        mParticleUser: MParticleUser?,
+        identityApiRequest: FilteredIdentityApiRequest?
+    ) {
+        setUserIdentity(mParticleUser)
     }
 
-    override fun logLtvIncrease(
-        valueIncreased: BigDecimal,
-        valueTotal: BigDecimal,
-        eventName: String,
-        contextInfo: Map<String, String>
-    ): List<ReportingMessage> = emptyList()
+    override fun onLoginCompleted(
+        mParticleUser: MParticleUser?,
+        identityApiRequest: FilteredIdentityApiRequest?
+    ) {
+        setUserIdentity(mParticleUser)
+    }
 
-    override fun logEvent(event: CommerceEvent): List<ReportingMessage> {
```

Comment:
I say that the CommerceListener was removed, therefore the ltvIncreaseEvent and logEvent (for commerce event) implementations were removed here too. Do we intentionally want no to listen those?

Reply:
Yes, We are not using `CommerceListener`.